src/main/java/io/jsonwebtoken/impl/ValidationHelper.java

```diff
@@ -0,0 +1,193 @@
+package io.jsonwebtoken.impl;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Clock;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Header;
+import io.jsonwebtoken.PrematureJwtException;
+
+/**
+ * This class contains static methods for validation of JWT claims.
+ */
+public class ValidationHelper {
```

Comment:
Upon further review of this class, it appears the most OO/pluggable way for this it to have a generic `Validator` interface and you could have multiple implementations, e.g. `ExpirationValidator` `PrematureValidator`, etc.  For example:

```java
public interface Validator<T> {
    public void validate(T value);
}
```

And the type used by the parser would be `Validator<Claims>`